src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -113,11 +114,11 @@ case class SignedTransaction (
   senderAddress: Address) {
 
   override def toString: String = {
-    s"""SignedTransaction {
-         |tx: $tx
-         |signature: $signature
-         |sender: ${Hex.toHexString(senderAddress.bytes.toArray)}
-         |}""".stripMargin
+    MoreObjects.toStringHelper(this)
+      .add("tx", tx)
+      .add("signature", signature)
+      .add("senderAddress", Hex.toHexString(senderAddress.bytes.toArray))
+      .toString
```

Comment:
What will this look like when printed? I was also looking into better formatting here, but was hoping for a more automatic solution, here are some ideas: https://stackoverflow.com/questions/15718506/scala-how-to-print-case-classes-like-pretty-printed-tree

Also, do we add the same for `Block(Body/Header)`, `Transaction`, etc.?